test/services/homework/copy-service.test.js

```diff
@@ -0,0 +1,106 @@
+const assert = require('assert');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['assert' is assigned a value but never used.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=4955622)